data-serving/scripts/export-data/country_export.sh

```diff
@@ -2,8 +2,13 @@
 
```

Comment:
Does this need to be a shell script? It would be easier to test, maintain, and modify as a Python script.
Especially if it's a shell script that calls a Python script.

Reply:
The previous implementation used mongoexport as a subprocess -- I have used a bash script to keep the dependencies in Python to zero, and as aws-cli has a simple interface that I'm planning on using in the full-export script, it keeps them similar.

Reply:
That makes sense for Lambda, to limit the layers, and I think for Batch moving away from bash makes more sense. Not a blocker by any means.